### DISCUSSION TITLE
When will lottie be supported?

### DISCUSSION THREAD

**Discussion by hxgqh** [arendst/Tasmota#22518] [Support request]
Supporting playing lottie json files(https://lottiefiles.com/). LVGL support [lottie](https://docs.lvgl.io/master/details/widgets/lottie.html).

**Comment by sfromis** [comment]
The file format Lottie? Or what are you talking about?

In general, the answer to "when will xxx be supported" is that it takes someone with the wish and willingness to do the work (if it is even feasible for Tasmota). You?

**Reply by hxgqh** [reply]
It means supporting playing lottie json files(https://lottiefiles.com/). LVGL support [lottie](https://docs.lvgl.io/master/details/widgets/lottie.html).

**Reply by s-hadinger** [reply]
I know. LVGL support Lottie **integration**. But the Lottie engine itself was not ported to Tasmota.

There are no plans to port Lottie to Tasmota unless somebody volunteers to add it.

**Comment by s-hadinger** [comment]
No plans unless somebody volunteers to add it

**Comment by hxgqh** [comment]
How to add member to module lv? I tried, but failed. Error: module 'lv' has no member 'lottie'

**Reply by s-hadinger** [reply]
Your question does not make sense without a complete context. Do you have a working Lottie runtime? On what platform are you running?
